sympy/vector/vector.py

```diff
@@ -344,6 +347,24 @@ def _div_helper(one, other):
         else:
             raise TypeError("Invalid division involving a vector")
 
+# The following is adapted from the matrices.expressions.matexpr file
+
+def get_postprocessor(cls):
+    def _postprocessor(expr):
+        vec_class = {Add: VectorAdd}[cls]
+        vectors = []
+        for term in expr.args:
+            if term.kind is VectorKind():
```

Comment:
If `VectorKind` is going to be a singleton (i.e. not parametrised) then it would be better to create it as a singleton like for NumberKind:
https://github.com/sympy/sympy/blob/06dbac029e785296ee3bbfb1001266806d94e98b/sympy/core/kind.py#L162

Reply:
VectorKind is now a parametrised Kind.

Reply:
Should this not check something like `if isinstance(term.kind, VectorKind)` because the kind might not be `VectorKind(NumberKind)`?

Reply:
Yes, it should. I'll change it now.